src/software/thunderscope/thunderscope.py

```diff
@@ -194,6 +194,10 @@ def __init__(
         # Setup the main window and load the requested tabs
         self.configure_layout(layout_path, load_blue, load_yellow, load_diagnostics)
 
+        self.keyboard_estop_shortcut = QtGui.QShortcut(
+            QtGui.QKeySequence(" "), self.window
+        )
```

Comment:
To make this work, I think you also need to add 
```python
self.keyboard_estop_shortcut.activated.connect(self....toggle_keyboard_estop)
```

Reply:
that's strange, I added this before during testing. must've accidentally reverted at some point